package.json

```diff
@@ -104,6 +104,7 @@
     "ts-helpers": "1.1.2",
     "ts-node": "3.0.4",
     "tslint": "5.2.0",
+    "tslint-jasmine-rules": "^1.5.1",
```

Comment:
Think we could remove the caret `^` and find the latest minor version that works?

Reply:
Caret is removed and `package.json` specifies `1.5.1` which is the latest version.